displayio/_group.py

```diff
@@ -108,38 +105,38 @@ def insert(self, index, layer):
         self._layers.insert(index, layer)
         self._layer_update(index)
 
-    def index(self, layer):
+    def index(self, layer: Union[Group, TileGrid]) -> int:
         """Returns the index of the first copy of layer.
         Raises ValueError if not found.
         """
         return self._layers.index(layer)
 
-    def pop(self, index=-1):
+    def pop(self, index=-1) -> Union[Group, TileGrid]:
         """Remove the ith item and return it."""
         self._removal_cleanup(index)
         return self._layers.pop(index)
 
-    def remove(self, layer):
+    def remove(self, layer) -> None:
         """Remove the first copy of layer. Raises ValueError
         if it is not present."""
         index = self.index(layer)
         self._layers.pop(index)
 
-    def __len__(self):
+    def __len__(self) -> int:
         """Returns the number of layers in a Group"""
         return len(self._layers)
 
-    def __getitem__(self, index):
+    def __getitem__(self, index) -> Union[Group, TileGrid]:
```

Comment:
could have `index: int` not sure if this is necessary for the builtin __getitem__ function though? maybe it's okay to leave it out.

Reply:
Yep, looks like I missed that one.